examples/python/telemetry_reader.py

```
@@ -86,15 +87,44 @@ class BufferHeader(ctypes.Structure):
+            "_pad_tail",
+            ctypes.c_char
+            * (DESTRUCTIVE_INTERFERENCE_SIZE - ctypes.sizeof(ctypes.c_size_t)),
+        ),  # match C++ compiler's tail padding: [264, 512)
     ]
 
 
+# Mirror the static_asserts in cyclic_buffer.tpp so a layout drift fails.
+# The pads are checked too since they are maintained by hand here, while
+# the C++ compiler derives them from alignas.
+if ctypes.sizeof(BufferHeader) != 2 * DESTRUCTIVE_INTERFERENCE_SIZE:
+    raise RuntimeError("BufferHeader layout drift: unexpected size")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)